### ISSUE TITLE
SLDS Linter Error: "No rules found within configuration" during lint command

### ISSUE THREAD

**Issue by krishnandusarkar** [salesforce-ux/slds-linter#191]
**Describe the bug**

When executing the command `npx @salesforce-ux/slds-linter@latest lint`, we encounter the following error:

`File processing failed: [File Path] - No rules found within configuration. Have you provided a "rules" property?`

**Configuration**

Please enter the following command in a terminal and copy/paste its output:

```bash
npx @salesforce-ux/slds-linter     
Usage: npx @salesforce-ux/slds-linter@latest [options] [command]

SLDS Linter CLI tool for linting styles and components

Options:
  -V, --version  output the version number
  -h, --help     display help for command

Commands:
  lint           Run both style and component linting
  report         Generate report from linting results
  emit           Emits the configuration files used by slds-linter cli
  help           display help for command 
```

**Comment by sf-nithappu** [comment]
@krishnandusarkar  We release new version of slds linter (0.2.20)[https://www.npmjs.com/package/@salesforce-ux/slds-linter/v/0.2.2] to address this issue.
